pkg/xds/envoy/tags/match.go

```diff
@@ -108,6 +109,28 @@ func (t Tags) String() string {
 	return strings.Join(pairs, ",")
 }
 
+func TagsFromTargetRef(targetRef common_api.TargetRef) (Tags, bool) {
```

Comment:
If the package is already called `tags`, probably makes sense to rename the function to `tags.FromTargetRef`